docs/components/store.md

```diff
@@ -104,6 +104,16 @@ Flags:
       --index-cache-size=250MB   Maximum size of items held in the index cache.
       --chunk-pool-size=2GB      Maximum size of concurrently allocatable bytes
                                  for chunks.
+      --grpc-sample-limit=50000000  
+                                 Maximum amount of samples returned via a single
+                                 Series call. 0 means no limit. NOTE: for
+                                 efficiency we take 120 as number of samples in
+                                 chunk, so the actual number of samples might be
+                                 lower, even though maximum could be hit. Cannot
+                                 be bigger than 120.
```

Comment:
I'm confused. You say it cannot be bigger than 120 but default is 50000000 ?

Reply:
It means that the upper-bound of samples per chunk is 120. Will move this part into parentheses in the former sentence: `NOTE: efficiency we take 120 as the number of samples in a chunk (it cannot be bigger than that), so the actual number of samples might be lower, even though maximum could be hit.`
Would it be clearer?